src/pages/ProductList/ProductList.scss

```
@@ -0,0 +1,121 @@
+.productList {
+  width: 1200px;
+  margin: auto;
+  .productListHeader {
+    border-bottom: 1px solid black;
+    padding-bottom: 40px;
+    padding-top: 80px;
+
+    .productListTitle {
+      font-size: 55px;
+      display: flex;
+      .totalQuantity {
+        font-size: 12px;
+        margin: 5px 0px 0px 14px;
+      }
+    }
+    .headerContentBox {
+      margin-top: 15px;
+      font-size: 15px;
+      text-align: left;
+      .headerContent {
+        height: 25px;
+      }
+    }
+  }
+  .filterBox {
+    display: flex;
+    justify-content: space-between;
+    padding-top: 20px;
+    .catAndDog {
+      .catBtn {
+        padding: 10px;
+        border: 1px solid #ffbb0f;
+        border-right: none;
+        border-radius: 2px;
+        font-size: 16px;
+        background-color: white;
+      }
+      .dogBtn {
+        padding: 10px;
+        border: 1px solid #ffbb0f;
+        border-radius: 2px;
+        font-size: 16px;
+        background-color: white;
+      }
+      .catBtn:hover {
+        background-color: red;
+        color: white;
+      }
+
+      .dogBtn:hover {
+        background-color: orange;
+        color: white;
+      }
+    }
+    .dropBox {
+      height: 33px;
+      font-size: 16px;
+      border: none;
+      background-color: white;
+      .dropBoxArrow {
+        padding-left: 8px;
+      }
+    }
+  }
+  .dropBoxListContainer {
+    position: relative;
+    .dropBoxList {
+      margin-left: 1081px;
+      margin-top: 6px;
+      padding-top: 15px;
+      padding-bottom: 15px;
+      padding-right: 15px;
```

```suggestion
      padding: 15px 15px 15px 0;
```
해당 스타일을 이렇게 적용할 수 있겠습니다!